### PR TITLE
tests: remove unused parameters

### PR DESCRIPTION
**Pull request by palango**
Removes some warnings related to unused parameters.

### CODE REVIEW

File: core/blockchain_test.go [hunk 1]
```diff
@@ -2413,7 +2413,7 @@ func testInsertKnownChainDataWithMerging(t *testing.T, typ string, mergeHeight i
 			}
 		}
 	} else {
-		inserter = func(blocks []*types.Block, receipts []types.Receipts) error {
+		inserter = func(blocks []*types.Block, _ []types.Receipts) error {
```
Comment (rjl493456442) [inline core/blockchain_test.go:2416]: Please revert this change. We typically prefer to have the param name left, even they are not used.

File: core/blockchain_test.go [hunk 2]
```diff
@@ -2239,7 +2239,7 @@ func testInsertKnownChainData(t *testing.T, typ string, scheme string) {
...
```
Comment (rjl493456442) [inline core/blockchain_test.go:2242]: Please revert this change. We typically prefer to have the param name left, even they are not used.

**Review by holiman — COMMENTED** [review]
IMO not useful change
